tsl/test/sql/continuous_aggs_multi.sql

```diff
@@ -248,3 +249,64 @@ refresh materialized view cagg_iia3;
 SELECT * FROM _timescaledb_catalog.continuous_aggs_materialization_invalidation_log order by 1;
 
 select * from cagg_iia3 order by 1;
+
+----TEST7 multiple continuous aggregates with real time aggregates test----
+create table foo (a integer, b integer, c integer);
```

Comment:
Nit: inconsistent use of non-capitalized and capitalized DML. Certainly not critical, but consistency is always nice.